linta-android/src/test/java/com/swvl/lint/checks/DuplicateResourceFilesDetectorTest.kt

```diff
@@ -297,6 +297,78 @@ class DuplicateResourceFilesDetectorTest {
+                    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:gravity="center|start"
+                        android:orientation="horizontal"
+                        android:paddingBottom="16dp">
+                        <!-- This is a button -->
+                        <Button
+                            android:id="@+id/addPassenger_button"
+                            style="@style/Widget.App.Button.Outlined.Primary"
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:paddingStart="20dp"
+                            android:paddingEnd="20dp"
+                            android:text="@string/travel_bookingConfirmation_addPassenger_button_title" />
+                            
+                        <!-- End of layout -->
+                        
+                        <LinearLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:gravity="center|start"
+                            android:orientation="horizontal">
+                            
+                            <!-- empty layout -->
+                            
+                        </LinearLayout>
+
+                    </LinearLayout>
+                    """
+                ).indented(),
+                xml(
+                    "res/layout/layout2.xml",
+                    """
+                    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:gravity="center|start"
+                        android:orientation="horizontal"
+                        android:paddingBottom="16dp">
+
+                        <Button
+                            android:id="@+id/addPassenger_button"
+                            style="@style/Widget.App.Button.Outlined.Primary"
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:paddingStart="20dp"
+                            android:paddingEnd="20dp"
+                            android:text="@string/travel_bookingConfirmation_addPassenger_button_title" />
+                            
+                            <LinearLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:gravity="center|start"
+                            android:orientation="horizontal">
+                            </LinearLayout>
```

Comment:
Same here. Indentation needs to be fixed.

```suggestion
                        </LinearLayout>
```

Reply:
done ✅